sqlglot/generator.py

```diff
@@ -538,6 +545,18 @@ def binary(self, expression, op, newline=False):
         sep = '\n' if newline else ' '
         return f"{self.sql(expression, 'this')}{sep}{op} {self.sql(expression, 'expression')}"
 
+    def function_fallback_sql(self, expression):
+        args = []
+        for arg_key in expression.arg_types.keys():
+            arg_value = expression.args.get(arg_key) or []
+            if not isinstance(arg_value, list):
```

Comment:
i feel like i've seen this pattern a bit, can we make a helper function called wrap_list(x: Any) -> List

Reply:
Added the `ensure_list` helper function and refactored accordingly.